lib/engine/game/g_1822/game.rb

```diff
@@ -727,11 +727,14 @@ def event_full_capitalisation!
         end
 
         def event_phase_revenue!
-          @log << '-- Event: P15-HR and P20-C&WR now closes and its money returned to the bank --'
+          @log << '-- Event: Phase revenue companies now close with money returned to the bank --'
           self.class::PRIVATE_PHASE_REVENUE.each do |company_id|
             company = @companies.find { |c| c.id == company_id }
             next if !company || company&.closed? || !@phase_revenue[company_id]
 
+            @log << "#{company.name} "\
+              "(#{format_currency(@phase_revenue[company_id].cash)}, #{company.owner.name}) "\
+              'closes'
```

Comment:
Why include cash and owner name? I don't believe any other private closing log message includes things like that. my personal preference is for consistency unless good reason.